modules/populateItems.js

```diff
@@ -112,9 +99,96 @@ export const populateItems = async () => {
     )
     .join('');
 
-  // calculate the prices of each item with quantity multiplier
-  const priceList = allItems.map((item) => item.price * item.quantity);
-  const totalPrice = priceList.reduce((a, b) => a + b, 0);
+  // create sticky footer sort options based on categories
+  const storeSectionData = generateStoreSectionData(allItems);
+  stickyQuickSortFooter.classList.remove('closed');
+
+  quickSortDiv.innerHTML = Object.keys(storeSectionData)
+    .sort()
+    .map(
+      (section) => `
+      <div class="itemGroupContainer" onclick="filterBySection('${section}')">
+        <div class="itemGroup ${section === selectedSection ? 'selected' : ''}
+        ${
+          storeSectionData[section].isPurchased ===
+          storeSectionData[section].total
+            ? 'completed'
+            : ''
+        }
+        ">
+          <p class="inCartNum">${storeSectionData[section].isPurchased}</p>
+          <p class="separator">/</p>
+          <p class="totalNum">${storeSectionData[section].total}</p>
+        </div>
+        <p class="itemGroupName">${capitalizeFirstLetter(section)}</p>
+      </div>
+    `
+    )
+    .join('');
+};
+
+const generateStoreSectionData = (items) => {
+  const storeSectionData = {};
+
+  items.forEach((item) => {
+    const section = item.section || 'Other';
+
+    // if the section isn't already recorded, add it
+    if (!storeSectionData[section]) {
+      storeSectionData[section] = { total: 0, isPurchased: 0 };
+    }
+    // always add it to the total count
+    storeSectionData[section].total += 1;
+    // determine if purchased
+    if (item.isPurchased) {
+      storeSectionData[section].isPurchased += 1;
+    }
+  });
+
+  return storeSectionData;
+};
+
+const filterBySection = (section) => {
+  if (selectedSection === section) {
+    selectedSection = null;
+  } else {
+    selectedSection = section;
+  }
+
+  populateItems();
+};
+
+// helper for section bubbles in footer
+const capitalizeFirstLetter = (string) => {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+};
+
+window.filterBySection = filterBySection;
+
+// helper to filter based on hideChecked and selected section
+const filterItems = (items, hideChecked, selectedSection) => {
+  let filteredItems = items;
+
+  if (hideChecked.checked) {
+    filteredItems = filteredItems.filter((item) => !item.isPurchased);
+  }
+
+  if (selectedSection) {
+    filteredItems = filteredItems.filter((item) =>
+      selectedSection === 'Other'
+        ? !item.section
+        : item.section === selectedSection
+    );
+  }
+
+  return filteredItems;
+};
+
+// sort items by section
+const sortItemsBySection = (items) => {
```

Comment:
This specifically maps to the "Sort by Section" in the options panel:

<img width="346" alt="image" src="https://github.com/user-attachments/assets/f71f1fd3-878c-4753-9dc4-1e5605888a0e">

I should better note this with a comment, and think about different names to avoid confusion.